src/main/java/com/github/timm/cucumber/generate/GenerateRunnersMojo.java

```diff
@@ -157,13 +175,11 @@ public void execute() throws MojoExecutionException {
             throw new MojoExecutionException("Features directory does not exist");
         }
 
-        final Collection<File> featureFiles =
-                        FileUtils.listFiles(featuresDirectory, new String[] {"feature"}, true);
+        final Collection<File> featureFiles = listFiles(featuresDirectory, new String[] {"feature"}, true);
+        final List<File> sortedFeatureFiles = new NameFileComparator().sort(new ArrayList<File>(featureFiles));
```

Comment:
Had to add the sort because there was no other way for the integration tests to pass on my system.